### ISSUE TITLE
Update protocol version for server intialize

### ISSUE THREAD

**Issue by tortmayr** [eclipse-glsp/glsp#706]
Currently is hardcoded to `0.9.0`:
```
2022-07-11 15:43:31,599422000 [pool-2-thread-1] DEBUG DefaultGLSPServer - Initializing server with the following params:
InitializeParameters [applicationId=fe1c055b-1242-492c-af85-cb845bf55c98, protocolVersion=0.9.0, args={ timestamp=Mon Jul 11 2022 15:43:30 GMT+0200 (Central European Summer Time),message=Custom Options Available } ]
```

Should be updated to 1.0.0 on both client and server

**Comment by planger** [comment]
https://github.com/eclipse-glsp/glsp/issues/782 indicates that we seem to have missed pulling up the VS Code integration. @xai can you please double-check?

**Comment by planger** [comment]
Closing this again, as we use https://github.com/eclipse-glsp/glsp/issues/782 to track this issue. Seems to be only affecting VS Code.

**Comment by xai** [comment]
I can reproduce the issue and verified that it is fixed by eclipse-glsp/glsp-vscode-integration#28 as @tortmayr just [noted](https://github.com/eclipse-glsp/glsp/issues/782#issuecomment-1313390277).
